src/App.js

```diff
@@ -11,33 +11,128 @@ class App extends React.Component {
       cardDeck: makeShuffledDeck(),
       // currCards holds the cards from the current round
       currCards: [],
+      // game start
```

Comment:
these kinds of comments are redundant if the code is self-documenting, aka good naming :)

Reply:
i understand but because of the comments are green so it helps me separate the lines better for me to read it better actually